Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -112,6 +136,7 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
         case string(String)
         case boolean(Bool)
         case numberArray([Double])
+        case stringArray([String])
```

Comment:
Does the style specification allow for other types to occur inside a literal array, such as an array of booleans or objects, or even heterogeneous arrays (which are allowed in JSON)? What if, instead of separate cases for each type that could occur inside an array, we have just a single `array` case that takes the evaluated type as an additional associated value?

Reply:
Yeah, i've considered doing this... It's just that modeling those kind of heterogenous arrays is hard. Open to suggestions here.